pkg/statistics/handle/util/util.go

```diff
@@ -94,6 +95,7 @@ func CallWithSCtx(pool SessionPool, f func(sctx sessionctx.Context) error, flags
 		}
 	}()
 	sctx := se.(sessionctx.Context)
+	sctx.GetSessionVars().ExplicitRequestSourceType = kvutil.ExplicitTypeStats
```

Comment:
Is it possible to get sessionvar from other internal SQLs, like from ddl module?